src/util/MongoBinary.js

```diff
@@ -24,7 +25,8 @@ export default class MongoBinary {
   static async getPath(opts?: MongoBinaryOpts = {}): Promise<string> {
     const defaultOptions = {
       downloadDir:
-        process.env?.MONGOMS_DOWNLOAD_DIR || path.resolve(os.homedir(), '.mongodb-binaries'),
```

Comment:
Like @nodkz suggested we should keep the fallback on **path.resolve(os.homedir(), '.mongodb-binaries')** until a new major version is bumped.